src/components/Carousel.js

```diff
@@ -398,6 +398,20 @@ export default class Carousel extends Component {
     this.resetInterval();
   };
 
+  simulatedEvent = event => {
+    const touch = event.changedTouches[0];
+    const simulatedEvent = document.createEvent('MouseEvent');
+
+    simulatedEvent.initMouseEvent(
```

Comment:
This method is deprecated. See https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent#Browser_compatibility

I'd suggest using the modern method of creating events: `new MouseEvent(typeArg, mouseEventInit);`
It accepts all params that you passed here, but as an object, which is more readable. Read more: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent

Reply:
Good find. Ok I'll write some tests and try the new method instead.